### ISSUE TITLE
Checking the session present flag on connect

### ISSUE THREAD

**Issue by danielnelson** [eclipse/paho.mqtt.golang#240]
My understanding is that when using persistent sessions I should check the "session present flag" on connect to determine if I need to create the subscriptions.  However, I don't see a way to retrieve this information, is it possible to get the value of this flag?

**Comment by alsm** [comment]
You're right this isn't currently exposed in the v3 client, I propose to add the field to the ConnectToken and you'll be able to query it from there.
